_includes/tutorials/anomaly-detection/ksql/markup/answer/join-and-windowing.adoc

```diff
@@ -0,0 +1,11 @@
+Create new stream with INNER JOIN between the table and stream.
```

Comment:
@awalther28 is the INNER JOIN is absolutely critical to the short answer for detecting anomalies?  Is it possible that only the WINDOWING is the key piece of the solution?  (In that, INNER JOIN is only required if the data isn't already joined...). Another motivation to remove the INNER JOIN out is to shorten the answer (and free up real estate on the site). I'm on the fence, just wanted to share this idea.

Reply:
I think the join is pretty critical, but imo a mention of it would suffice. @ybyzek what are your thoughts on something like

> Given that transaction events are joined with table reference data, use WINDOWING to group anomalous transactions. //insert windowing render here//

Reply:
SGTM!